addons/mail/models/mail_activity.py

```diff
@@ -480,24 +490,18 @@ def action_notify(self):
             model_description = activity.env['ir.model']._get(activity.res_model).display_name
             body = activity.env['ir.qweb']._render(
                 'mail.message_activity_assigned',
-                dict(
-                    activity=activity,
-                    model_description=model_description,
-                    access_link=activity.env['mail.thread']._notify_get_action_link('view', model=activity.res_model, res_id=activity.res_id),
-                ),
+                dict(activity=activity, model_description=model_description, is_html_empty=is_html_empty),
```

Comment:
You can keep a mutli-line approach, lessen diff when adding stuff in the values dict (as only the new line will be greened in the diff), like
```
body = activity.env['ir.qweb']._render(
    'mail.message_activity_assigned',
    {
        'activity': activity,
        'model_description': model_description,
        'is_html_empty': is_html_empty,
    },
    ...
```

Reply:
Yes of course, it easier to review after.